examples/intel_extension_for_pytorch/README.md

```diff
@@ -0,0 +1,179 @@
+torch.jit.save(model, 'bert_int8_jit.pt')
+```
+
+#### ResNet50 
+
+```
+import intel_extension_for_pytorch as ipex
+import torchvision.models as models
+import torch
+import torch.fx.experimental.optimization as optimization
+from copy import deepcopy
+
+
+model = models.resnet50(pretrained=True)
+model = model.eval()
+
+C, H, W = 3, 224, 224
+dummy_tensor = torch.randn(1, C, H, W).contiguous(memory_format=torch.channels_last)
+jit_inputs = (dummy_tensor)
+conf = ipex.quantization.QuantConf(qscheme=torch.per_tensor_symmetric)
+
+n_iter = 100 
+with torch.no_grad():
+	for i in range(n_iter):
+		with ipex.quantization.calibrate(conf):
+			model(dummy_tensor)
+
+model = ipex.quantization.convert(model, conf, jit_inputs)
+torch.jit.save(model, 'rn50_int8_jit.pt')
+```
+### 2. Creating a Model Archive 
+Once the serialized file ( `.pt`) is created, it can be used with `torch-model-archiver` as ususal. Use the following command to package the model.  
+```
+torch-model-archiver --model-name rn50_ipex_int8 --version 1.0 --serialized-file rn50_int8_jit.pt --handler image_classifier 
+```
+### 3. Start Torchserve to serve the model 
+Make sure to set `ipex_enable=true` in `config.properties`. Use the following command to start Torchserve with IPEX. 
+```
+torchserve --start --ncs --model-store model_store --ts-config config.properties
+```
+
+### 4. Registering and Deploying model 
+Registering and deploying the model follows the same steps shown [here](https://pytorch.org/serve/use_cases.html). 
+
+## Torchserve with Launcher
+Launcher is a script to automate the process of tunining configuration setting on intel hardware to boost performance. Tuning configurations such as OMP_NUM_THREADS, thread affininty, memory allocator can have a dramatic effect on performance. Please refer to [here](https://github.com/intel/intel-extension-for-pytorch/blob/master/docs/tutorials/performance_tuning/tuning_guide.md) and [here](https://github.com/intel/intel-extension-for-pytorch/blob/master/docs/tutorials/performance_tuning/tuning_guide.md) for details on performance tuning with launcher. 
+
+All it needs to be done to use Torchserve with launcher is to set its configuration in `config.properties`.
+
+
+Add the following lines in `config.properties` to use launcher with its default configuration. 
+```
+ipex_enable=true
+cpu_launcher_enable=true
+```
+
+Launcher by default uses `numactl` if its installed to ensure socket is pinned and thus memory is allocated from local numa node. To use launcher without numactl, add the following lines in `config.properties`.
+```
+ipex_enable=true
+cpu_launcher_enable=true
+cpu_launcher_args=--disable_numactl
+```
+
+Launcher by default uses only non-hyperthreaded cores if hyperthreading is present to avoid core compute resource sharing. To use launcher with all cores, both physical and logical, add the following lines in `config.properties`.  
+```
+ipex_enable=true
+cpu_launcher_enable=true
+cpu_launcher_args=--use_logical_core
```

Comment:
could we add a longer list of `cpu_launcher_args` which ones are the most important for users to know about?

Reply:
Yes - we're currently testing performance with and without launcher to recommend configurations to users

Reply:
also it would be helpful to show how to pass multiple options to the  "cpu_launcher_args"

Reply:
yes- here is a simple example fyi: `cpu_launcher_args=--user_logical_core --disable_numactl`; will update the README as well